pkgs/applications/misc/ArchiSteamFarm/web-ui/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib, pkgs, fetchFromGitHub, nodejs, stdenv, ArchiSteamFarm, ... }:
+
+let
+  nodePackages = import ./node-composition.nix {
+    inherit pkgs nodejs;
+    inherit (stdenv.hostPlatform) system;
+  };
+
+  src = fetchFromGitHub {
+    owner = "JustArchiNET";
+    repo = "ASF-ui";
+    # updated by the update script
+    # this is always the commit that should be used with asf-ui from the latest asf version
+    rev = "cff26d6ebc65c66497a5d68543c695e442f20240";
+    sha256 = "0nra7xj0lq6dw1312fa956ivsg9df08pbxmdmws9ilvq2nzf55v1";
+  };
```

Comment:
Wouldn't it be easier to just clone the submodule in ASF and then point this to the directory?

Reply:
We already need an update script, and the logic required for this isn't complex. So why not keep the build closure as small as possible.

Reply:
I don't think we safe anything here. Having two sources which are in sum as big as the combined src does not keep the size down.